src/ducks/Signals/chart/VisualBacktestChart.js

```diff
@@ -0,0 +1,170 @@
+import React from 'react'
+import {
+  ResponsiveContainer,
+  ComposedChart,
+  XAxis,
+  YAxis,
+  Tooltip,
+  ReferenceDot,
+  ReferenceLine
+} from 'recharts'
+import cx from 'classnames'
+import { generateMetricsMarkup } from './../../SANCharts/utils'
+import { formatNumber } from './../../../utils/formatting'
+import { getDateFormats, getTimeFormats } from '../../../utils/dates'
+import chartStyles from './../../SANCharts/Chart.module.scss'
+import sharedStyles from './../../SANCharts/ChartPage.module.scss'
+import styles from './SignalPreview.module.scss'
+
+export const getDataKeys = (signal = {}) => {
+  if (signal.active_addresses) {
+    return { metric: 'dailyActiveAddresses', signal: 'active_addresses' }
+  }
+  if (signal.price) return { metric: 'priceUsd', signal: 'price' }
```

Comment:
Each time when I see "if" and return without brackets, I am feeling possible error. But here I see an inconsistency, 2 "if's" one with brackets, one is without